src/components/new-request/search.vue

```diff
@@ -311,8 +314,10 @@ export default class NewSearch extends Mixins(CommonMixin) {
 
   // Local properties
   corpNumValid = true
-  readonly corpOnlineLink = 'https://www.corporateonline.gov.bc.ca/'
   locationDisabled = false
+  selectedCompanyType = ''
```

Comment:
This declares the type as string. It should be CompanyType instead. (You'll probably need to init as null.)